gradoop-temporal/src/main/java/org/gradoop/temporal/model/impl/functions/predicates/Overlaps.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.temporal.model.impl.functions.predicates;
+
+import org.gradoop.temporal.model.api.functions.TemporalPredicate;
+
+import java.time.LocalDateTime;
+
+import static org.gradoop.temporal.util.TimeFormatConversion.toEpochMilli;
+
+/**
+ * Implementation of the <b>Overlaps</b> temporal predicate.
+ * Given a certain time interval, this predicates matches all intervals that overlap with that interval.
+ * <p>
+ * Predicate: max(from, queryFrom) < min(to, queryTo)
```

Comment:
Please put this in a `code` block to avoid javadoc errors (especially with the `<` character)